modules/olmv1-about-target-versions.adoc

```diff
@@ -0,0 +1,116 @@
+= About target versions in OLM 1.0
+
+In Operator Lifecycle Manager 1.0, cluster administrators set the target version of an Operator in the Operator's custom resource (CR).
+
+If the Operator's version is not specified in the CR, then OLM 1.0 defaults to the latest stable version in the catalog as the target version. When updates to the Operator are published to the catalog, the Operator automatically updates to the latest stable version.
+
+.Example CR without a specified target version
+[source,yaml]
+----
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: quay-example
+spec:
+  packageName: quay-operator
+----
+
+If you specify the Operator's target version in the CR, OLM 1.0 installs the specified version. When the target version is specified in the Operator's CR, OLM 1.0 does not change the target version.
+
+.Example CR with the target version specified
+[source,yaml]
+----
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: quay-example
+spec:
+  packageName: quay-operator
+  channel: stable-3.8
+  version: 3.8.12
+----
+
+Edit the target version to change the installed version of an Operator. Provided that the update is supported by the skip ranges defined in the catalog, you can update across channels and to an earlier version an Operator.
+
+.Example CR with an updated target version
+[source,yaml]
+----
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: quay-example
+spec:
+  packageName: quay-operator
+  channel: stable-3.9
+  version: 3.9.1
+----
+
+You can inspect an Operator's catalog contents, including supported versions and skip ranges, by running the following command:
+
+.Command syntax
+[source,terminal]
+----
+$ oc get package <catalog_name>-<operator_name> -o yaml
+----
+
+After a CR is created or updated, run the following command to create or configure the Operator:
+
+.Command syntax
+[source,terminal]
+----
+$ oc apply -f <extension_name>.yaml
+----
+
+[NOTE]
+====
+If you make a mistake and update to a target version or channel that does not exist, you can run the following command to check the status of your Operator:
+
+[source,terminal]
+----
+$ oc get operators.operators.operatorframework.io <operator_name> -o yaml
+----
+
+.Example output
+[source,text]
+----
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"operators.operatorframework.io/v1alpha1","kind":"Operator","metadata":{"anno
+tations":{},"name":"quay-example"},"spec":{"channel":"stable-3.9","packageName":"quay-operator","
+version":"999.99.9"}}
+  creationTimestamp: "2023-10-12T20:20:40Z"
+  generation: 2
+  name: quay-example
+  resourceVersion: "45505"
+  uid: 01ba4dd6-4eaa-4e62-bff1-233770915f7b
+spec:
+  channel: stable-3.9
+  packageName: quay-operator
+  version: 999.99.9
+status:
```

Comment:
Would it help to use code callouts to explain what the different statuses/conditions mean?